### PR TITLE
Restore a working linking setup for glimmer-vm packages

### PR DESCRIPTION
**Pull request by chancancode**
See https://github.com/glimmerjs/glimmer-vm/pull/1523

### CODE REVIEW

**Review by NullVoxPopuli — APPROVED** [review]

File: bin/link-glimmer-vm-packages.mjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const packageJsonPath = new URL('../package.json', import.meta.url).pathname;
+const packageJson = JSON.parse(await readFileSync(packageJsonPath, { encoding: 'utf8' }));
+
+function shouldLink(dep) {
+  return dep.startsWith('@glimmer/') && dep !== '@glimmer/component' && dep !== '@glimmer/env';
+}
+
+for (const [dep] of Object.entries(packageJson.dependencies)) {
+  if (shouldLink(dep)) {
+    await execa('pnpm', ['link', '--global', dep], { stdio: 'inherit' });
```
[the remark this answers is not in the document]
Comment (chancancode) [reply bin/link-glimmer-vm-packages.mjs:13]: > This'll require folks to not use volta locally (oh no!) lol (for the better).

Why is that? I don't _think_ I ran into issues 🤔

**Comment by chancancode** [comment]
Need to look into a couple of build issues/warnings

File: bin/link-glimmer-vm-packages.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (chancancode) [reply bin/link-glimmer-vm-packages.mjs:13]: 🤔 somehow I didn't encounter that problem

Anyway, we could also make it take an argument and point to the `linkable-dist` if folks run into issues, the global part isn't strictly required as long as the two sides match

**Comment by chancancode** [comment]
Fixed the build issue, this should work now

File: bin/link-glimmer-vm-packages.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (chancancode) [reply bin/link-glimmer-vm-packages.mjs:13]: I think this didn't end up being a problem (not directly, at least)

[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply bin/link-glimmer-vm-packages.mjs:13]: confirm -- I had another issue related to my local config

**Pull request merged**